### PR TITLE
Add Windows packages

### PR DESCRIPTION
**Pull request by davewalter**
This PR adds Windows packages for v6, v7, and v8. It also updates the test manifest to use Bionic stemcells for Linux testing.

### CODE REVIEW

**Comment by rkoster** [comment]
This PR should be reviewed by an approver from the Runtime Interfaces working group, as discussed here: https://cloudfoundry.slack.com/archives/C02HPPYQ2/p1648073809996909?thread_ts=1647644255.599029&cid=C02HPPYQ2

**Comment by davewalter** [comment]
Is it possible for someone to review this?

**Comment by jdgonzaleza** [comment]
v6 will be out of support in the near future, do we still want to keep it ? I think that this was something we've discussed with @a-b and @reedr3

File: ci/pipeline.yml [hunk 1]
```diff
@@ -36,20 +36,41 @@ resources:
     regexp: releases/v(.*)/cf-cli_(.*)_linux_x86-64.tgz
     region_name: us-west-1
 
+- name: v6-cli-binary-windows-64
+  type: s3
+  source:
+    bucket: cf-cli-releases
+    regexp: releases/v(.*)/cf-cli_(.*)_winx64.tgz
```
Comment (jdgonzaleza) [inline ci/pipeline.yml:43]: Just checked the s3 buckets and windows binaries are .zip files

**Review by jdgonzaleza — COMMENTED** [review]
Please  revisit the regex in the s3 bucket resource for windows.

File: ci/pipeline.yml (hunk 1 above)
Comment (davewalter) [reply ci/pipeline.yml]: Thanks for catching this @jdgonzaleza.

**Pull request closed without merging**